.build/msi/google-cloudops-opentelemetry-collector.wxs

```diff
@@ -0,0 +1,53 @@
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
```

Comment:
Slightly restructured this file so that we install to the same directory as if installed via GooGet. Github should have picked it up as a rename.